### PR TITLE
Wagtail 5.2

### PR DESCRIPTION
**Pull request by katdom13**
## [Wagtail 5.2 release notes](https://docs.wagtail.org/en/stable/releases/5.2.html)

Changes:
- Add Wagtail 5.2 and Python 3.12 in test matrices, update CHANGELOG.md

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -2,16 +2,18 @@
 min_version = 4.0
 
 envlist =
-    python{3.8,3.9,3.10}-django{3.2}-wagtail{4.1,5.0,5.1}
-    python{3.9,3.10,3.11}-django{4.1}-wagtail{4.1,5.0,5.1}
-    python{3.10,3.11}-django{4.2}-wagtail{5.0,5.1}
+    python{3.8,3.9,3.10}-django{3.2}-wagtail{4.1,5.0,5.1,5.2}
```
Comment (zerolab) [inline tox.ini:5]: can you please drop 5.0 from the test matrix and deps. it is officially EOL - https://endoflife.date/wagtail

Comment (katdom13) [reply tox.ini]: Hi @zerolab ,
I removed it here and updated the one on README.md:
https://github.com/torchbox/wagtail-footnotes/pull/62/commits/6e94fa53269af52a1e27ac767eb839efa639a4e8

Thanks for checking!

File: wagtail_footnotes/blocks.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def replace_tag(match):
         return mark_safe(FIND_FOOTNOTE_TAG.sub(replace_tag, html))  # noqa: S308
 
     def render(self, value, context=None):
-        if not self.get_template(context=context):
+        if not self.get_template(value=value, context=context):
```
Comment (zerolab) [inline wagtail_footnotes/blocks.py:56]: I'm pretty sure this was only added in 5.2

See https://github.com/wagtail/wagtail/blob/stable/5.1.x/wagtail/blocks/base.py#L210 vs https://github.com/wagtail/wagtail/blob/stable/5.2.x/wagtail/blocks/base.py#L213

Comment (katdom13) [reply wagtail_footnotes/blocks.py]: @zerolab ,
Thanks, I forgot to put the upgrade consideration under a conditional.
Please see [latest commit](wagtail_footnotes/blocks.py).

**Comment by katdom13** [comment]
Superseded by https://github.com/torchbox/wagtail-footnotes/pull/63

**Pull request closed without merging**